modules/apps/jaggery/emm/src/emm/units/platform-configuration/public/js/platform-configuration.js

```diff
@@ -427,7 +427,7 @@ $(document).ready(function () {
             $(errorMsg).text("Monitoring frequency is a required field. It cannot be empty.");
             $(errorMsgWrapper).removeClass("hidden");
         } else if (!isPositiveInteger(notifierFrequency)) {
-            $(errorMsg).text("Provided monitoring frequency is invalid. Please check.");
+            $(errorMsg).text("Provided monitoring frequency is invalid. ");
```

Comment:
Recommended fixes  can be found in PR[1]

[1]. https://github.com/wso2/product-mdm/pull/158